src/Controller/Admin/QrCrudController.php

```diff
@@ -37,6 +42,17 @@ public static function getEntityFqcn(): string
 
     public function createEntity(string $entityFqcn): Qr
     {
+        yield IdField::new('id', 'ID')
```

Comment:
i don't think yield belongs here. It turns the class into a generator and we don't want that.